tests/integration/integration-test.py

```diff
@@ -14,8 +17,8 @@ def setup(cls, config_path, openapi_path):
             cls.session = utils.setup_session(config)
             cls.test_cases = config['test_cases']
 
-        with open(openapi_path) as openapi_file:
-            cls.openapi = yaml.load(openapi_file)
+        parser = ResolvingParser(openapi_path, backend='flex')
```

Comment:
I think `backend` should be configurable since `flex` won't be supported for openapi 3.